nemo_curator/utils/distributed_utils.py

```diff
@@ -633,3 +636,16 @@ def get_network_interfaces() -> List[str]:
         A list of all valid network interfaces on a machine
     """
     return list(psutil.net_if_addrs().keys())
+
+
+def get_gpu_memory_info() -> List[dict]:
+    """
+    Gets the total and available memory for each GPU on the machine
+
+    Returns:
+        A list of dictionaries containing the total and available memory for each GPU on the machine
+    """
+    client = get_current_client()
+    if client is None:
+        return {}
```

Comment:
Nit: Return an empty list instead of an empty dict (assuming the return type is accurate)

Reply:
Fixed this : https://github.com/NVIDIA/NeMo-Curator/pull/168/commits/e889f51a9f8f1fbe77f9ce256cb378c0f3083e32